File: GameRanks/src/main/java/GameRanks/GameRanks/clientStruct/page/PublisherPageStruct.java
package GameRanks.GameRanks.clientStruct.page;

import GameRanks.GameRanks.clientStruct.element.PublisherStruct;
import GameRanks.GameRanks.serverStruct.StatisticStruct;
import lombok.AllArgsConstructor;
import lombok.Data;

@Data
@AllArgsConstructor
public class PublisherPageStruct {
    private PublisherStruct publisherStruct;
    private StatisticStruct statisticStruct;
}

File: GameRanksClient/src/app/models/serverStructs/page/PublisherPageStruct.ts
import {PublisherStruct} from "../element/PublisherStruct";
import {StatisticStruct} from "../element/StatisticStruct";

export class PublisherPageStruct{
	publisherStruct: PublisherStruct;
	statisticStruct: StatisticStruct;

	constructor(publisherStruct: PublisherStruct, statisticStruct: StatisticStruct){
		this.publisherStruct = publisherStruct;
		this.statisticStruct = statisticStruct;
	}
}
